opengrok-indexer/src/main/java/org/opengrok/indexer/authorization/AuthorizationFramework.java

```diff
@@ -763,6 +763,11 @@ private boolean checkAll(HttpServletRequest request, String cache, Nameable enti
             return true;
         }
 
+        if (entity == null) {
+            LOGGER.log(Level.WARNING, "entity was null for request {}", request);
```

Comment:
maybe an entity class (getSimpleName) would also be a benefit for debuging

Reply:
and did you try what request.toString returns? I guess it's not any valueable info just the reference number.

Reply:
will use `request.getRequestURI()`

Reply:
or maybe `request.getParameterMap()` would be more useful.

Reply:
may be. a little bit too long eventually for searches

Reply:
yes, however might be crucial for figuring out what went wrong.